website-next/components/header.js

```diff
@@ -7,6 +7,11 @@ export function Header() {
       <title>Utopia</title>
       <meta name='viewport' content='width=device-width,initial-scale=1,shrink-to-fit=no' />
 
+      {/* preload fonts */}
+      <link rel="preload" href="/fonts/RecklessNeue-Book.woff2" crossorigin="anonymous" as="font" type="font/woff2" />
```

Comment:
what about the other versions of Reckless?

Reply:
those are not used yet